modules/Initialize.pm

```diff
@@ -145,6 +153,23 @@ sub load_gene_set {
   return \%stats;
 }
 
+sub _check_biotypes {
+  my ($db) = @_;
+
+  my %known_types = ();
+  for my $name (keys %BIOTYPE) {
+    %known_types = (%known_types, %{$BIOTYPE{$name}});
+  }
+  my @unsupported = ();
+  for my $biotype ($db->types) {
+    $biotype =~ s/:.+$//;
+    if (not exists $known_types{$biotype}) {
+      push @unsupported, $biotype;
+    }
+  }
+  die("Unknown biotypes: " . join(", ", @unsupported)) if @unsupported;
```

Comment:
Just running the script on test dataset, do we want to kill the script here or return a warning if there are `Unknown biotypes` ?

Reply:
We want an explicit rule for those biotypes: either supported, or skipped.
The script should stop early (die) if it finds unknown biotypes so we can decide what to do without running the whole thing.